### DISCUSSION TITLE
/mnt/docker-aio-config is not a mountpoint. Cannot proceed!

### DISCUSSION THREAD

**Discussion by waleedsworld** [nextcloud/all-in-one#2739] [Questions] [answered]
Digest: sha256:c520698cafd5d0316e22afdc336ab55e756434f16b295e62d280134ad0ab8fc3
Status: Downloaded newer image for nextcloud/all-in-one:latest
/mnt/docker-aio-config is not a mountpoint. Cannot proceed!

**Comment by szaimen** [comment]
Hi, can you post the output of `sudo docker inspect nextcloud-aio-mastercontainer` here?

**Reply by waleedsworld** [reply]
sorry for the late reply honestly I thought nobody would reply like many other so I shifted to owncloud and deleted nextcloud but since you have replied I am back to nextcloud but now I face another error 
docker: Error response from daemon: manifest for nextcloud/all-in-one:latests not found: manifest unknown: manifest unknown.
See 'docker run --help'.

**Reply by szaimen** [reply]
I think I replied even the same day so wondering why you thought that no one would reply?

**Reply by Zoey2936** [reply]
you have a typo in the image tag, you need to use :latest, but you wrote :latests

**Reply by waleedsworld** [reply]
I am really sorry I dont check github that often but mostly I do not get many replies
I have fixed the typo but still I am getting the same error
`admin@truenas[/mnt/My pool/next-cloud]$ sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/My pool/next-cloud/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest
Unable to find image 'pool/next-cloud/docker-aio-config:latest' locally
docker: Error response from daemon: Head "https://registry-1.docker.io/v2/pool/next-cloud/docker-aio-config/manifests/latest": unknown: invalid repository name.
See 'docker run --help'.`

**Reply by szaimen** [reply]
Hi, please copy and paste the exact commabd from step 3 of the readme without modifying anything: https://github.com/nextcloud/all-in-one#how-to-use-this

**Reply by waleedsworld** [reply]
it will give me a permission error because I am on truenas here
`admin@truenas[~]$ # For Linux and without a web server or reverse proxy (like Apache, Nginx, Cloudflare Tunnel and else) already in place:
sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest
zsh: number expected
[sudo] password for admin:
docker: Error response from daemon: invalid volume specification: 'nextcloud_aio_mastercontainer:/mnt/docker-aio-config': 'nextcloud_aio_mastercontainer' 'path' not allowed to be mounted.
See 'docker run --help'.`

**Reply by szaimen** [reply]
Is this truenas scale?

**Reply by waleedsworld** [reply]
yes I have provided a complete description down below

**Comment by waleedsworld** [comment]
I am on trueNas my drive is at /mnt/ on which I want to run nextcloud
it gives me an error first
`Digest: sha256:c520698cafd5d0316e22afdc336ab55e756434f16b295e62d280134ad0ab8fc3
Status: Downloaded newer image for nextcloud/all-in-one:latest
/mnt/docker-aio-config is not a mountpoint. Cannot proceed!`
then
`docker: Error response from daemon: manifest for nextcloud/all-in-one:latests not found: manifest unknown: manifest unknown.
See 'docker run --help'.`
now
`admin@truenas[/mnt/My pool/next-cloud]$ sudo docker run \ --sig-proxy=false \ --name nextcloud-aio-mastercontainer \ --restart always \ --publish 80:80 \ --publish 8080:8080 \ --publish 8443:8443 \ --volume nextcloud_aio_mastercontainer:/mnt/My pool/next-cloud/docker-aio-config \ --volume /var/run/docker.sock:/var/run/docker.sock:ro \ nextcloud/all-in-one:latest Unable to find image 'pool/next-cloud/docker-aio-config:latest' locally docker: Error response from daemon: Head "https://registry-1.docker.io/v2/pool/next-cloud/docker-aio-config/manifests/latest": unknown: invalid repository name. See 'docker run --help'.`

**Comment by waleedsworld** [comment]
yes

**Comment by szaimen** [comment, marked as answer]
See https://github.com/nextcloud/all-in-one#can-i-run-aio-on-truenas-scale

**Comment by waleedsworld** [comment]
`admin@truenas[~]$ helm install my-release nextcloud-aio/nextcloud-aio-helm-chart -f values.yaml
Error: INSTALLATION FAILED: Kubernetes cluster unreachable: the server has asked for the client to provide credentials
admin@truenas[~]$ sudo helm install my-release nextcloud-aio/nextcloud-aio-helm-chart -f values.yaml
[sudo] password for admin:
Error: INSTALLATION FAILED: failed to download "nextcloud-aio/nextcloud-aio-helm-chart"`

**Reply by szaimen** [reply]
Hi, you cannot install the helm chart directly as mentioned in the swction about trunas scale. You need to install portainer first and then install the helm chart via portainer or install AIO in a VM.

**Comment by waleedsworld** [comment]
when I try to install it on my mac
I get the error
*Cannot connect to the docker socket. Cannot proceed.
`sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 81:80 \
--publish 8082:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock.raw:/var/run/docker.sock:ro \
--env NEXTCLOUD_DATADIR="/Users/oyehroonn/Desktop/waleed/nextcloud" \
nextcloud/all-in-one:latest
Cannot connect to the docker socket. Cannot proceed.`

I searched and found where docker.socket is for colima when I used that I got the error
*Docker socket is not readable by the www-data user. Cannot continue.
`sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 81:80 \
--publish 8082:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /Users/oyehroonn/.colima/default/docker.sock:/var/run/docker.sock:ro \
--env NEXTCLOUD_DATADIR="/Users/oyehroonn/Desktop/waleed/nextcloud" \
nextcloud/all-in-one:latest
Trying to fix docker.sock permissions internally...
Adding internal www-data to group dialout
Docker socket is not readable by the www-data user. Cannot continue.`

some more information
Software:
    System Software Overview:

      System Version: macOS 12.0.1 (21A559)
      Kernel Version: Darwin 21.1.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Computer Name: Haroon’s MacBook Pro
      User Name: Haroon Ajmal (oyehroonn)
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 3 days 2:55

docker installed using colima
colima status output:
INFO[0000] colima is running using QEMU
INFO[0000] arch: aarch64
INFO[0000] runtime: docker
INFO[0000] mountType: sshfs
INFO[0000] socket: unix:///Users/oyehroonn/.colima/default/docker.sock

docker ps
`CONTAINER ID   IMAGE                               COMMAND                  CREATED         STATUS                          PORTS
                              NAMES
25ee3dad1e52   nextcloud/all-in-one:latest         "/start.sh /usr/bin/…"   3 minutes ago   Restarting (1) 14 seconds ago
                              nextcloud-aio-mastercontainer
e42585e7c52a   amrka/ultimate-android              "/usr/local/bin/mvn-…"   43 hours ago    Up 43 hours                     0.0.0.0:5900->5900/tcp, :::5900->5900/tcp             androidImage
57086363f231   owncloud/server:10.12               "/usr/bin/entrypoint…"   3 days ago      Up 3 days (healthy)             0.0.0.0:8081->8080/tcp, :::8081->8080/tcp             owncloud_server
16a606ac86a0   mariadb:10.6                        "docker-entrypoint.s…"   3 days ago      Up 3 days (healthy)             3306/tcp
                              owncloud_mariadb
8770e4be8059   redis:6                             "docker-entrypoint.s…"   3 days ago      Up 3 days (healthy)             6379/tcp
                              owncloud_redis
4a8c6fe9df92   linuxserver/webtop                  "/init"                  7 days ago      Up 3 days                       0.0.0.0:3000->3000/tcp, :::3000->3000/tcp, 3001/tcp   webtop
217330a46421   linuxserver/jackett                 "/init"                  7 days ago      Up 3 days                       0.0.0.0:9117->9117/tcp, :::9117->9117/tcp             jackett
18c84ac6a93c   quay.io/go-skynet/local-ai:latest   "/build/entrypoint.s…"   10 days ago     Up 3 days (healthy)             0.0.0.0:8080->8080/tcp, :::8080->8080/tcp             localai-api-1`
